### ISSUE TITLE
Search Notes

### ISSUE THREAD

**Issue by stefan-niedermann** [owncloud/notes#176]
Feature Request: Search / Filter Notes
If one has very much Notes it would be very useful to search for notes. Maybe registering a searchprovider so the OC Search (top right) would work? would be great.
Or as a small step: An input field as second list point (under + New Note-Button)

CC @jancborchardt

**Comment by jancborchardt** [comment]
Yep, that would indeed be good. Are you interested in implementing that? :)

cc @Henni @LukasReschke @owncloud/designers for info:
We should register a searchprovider, always. Using the API is important, the placement in the interface is a different discussion.

The placement is a discussion we should also have though. For Notes indeed it makes sense to have it as first entry in the left navigation, as it does for some other apps. We could try moving it in there via CSS to just play around and see where it makes sense.

**Comment by stefan-niedermann** [comment]
i want to invest my time for the android client ;)

**Comment by jancborchardt** [comment]
Sure, I don’t want to take your focus from that. :)
